### PR TITLE
Platform upgrade

### PR DESCRIPTION
**Pull request by godind**
Update to Node 14, Angular 12 plus a bunch of dependencies. Minor AP UI fixes to match upcoming n2k-SignalK and SignalK-Autopilot plugins and fully enable the Kip autopilot control.

### CODE REVIEW

**Comment by godind** [comment]
Hi @mxtommy,
Could you review and approve please. Authors can't self approve PRs.

Thanks

File: src/app/alarm-menu/alarm-menu.component.scss [hunk 1]
```diff
@@ -1,6 +1,3 @@
-@import '~@angular/material/theming';
```
Comment (VibroAxe) [inline src/app/alarm-menu/alarm-menu.component.scss:1]: Does this need `@use '~@angular/material' as mat;` as per further down or is material just not used in alarm-menu?

File: src/app/svg-wind/svg-wind.component.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // file used by various SVG components
+@use '~@angular/material' as mat;
```
Comment (VibroAxe) [inline src/app/svg-wind/svg-wind.component.scss:2]: this should probably come after the `do not delete` comment, or remove the comment below?

File: src/index.html [hunk 3]
```diff
@@ -12,6 +12,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="stylesheet" href="assets/fontawesome6/css/all.min.css">
+  <!-- <link rel="stylesheet" href="assets/bootstrap.min.css"> -->
```
Comment (VibroAxe) [inline src/index.html:15]: what does this actually do/why? Doesn't appear to be in master/index.html? Appears to be a redundant line

**Review by VibroAxe — COMMENTED** [review]
Looks good to me, couple of minor questions/queries. Code tested and operates fine on both standalone `ng serve` and signalK integrated `npm link`

@mxtommy I'd support merging this.

**Comment by VibroAxe** [comment]
@godind should also probably note that I think this superceeds all(?) the dependabot merge requests also open?

**Comment by VibroAxe** [comment]
Also closes #111

File: src/index.html (hunk 3 above)
Comment (godind) [reply src/index.html]: Good catch. Old leftover

**Review by godind — COMMENTED** [review]
Minor cleanups

**Review by godind — COMMENTED** [review]
@VibroAxe I'm not experienced with branching, Reviews, Approvals, etc. as you can see. I think if you approve a review I can merge. Else we will need Tommy.

**Comment by mxtommy** [comment]
> > @VibroAxe I'm not experienced with branching, Reviews, Approvals, etc. as you can see. I think if you approve a review I can merge. Else we will need Tommy.
> 
> Unfortuantely on this repo it looks like two of us would need write access. Lets see if @mxtommy comes back, if not, I can probably resubmit your PR such that you can approve it. Give it a week?
> 
> image
> 
> Do you have access to the npm repo or do we need @mxtommy for that as well?

I think I can add godind to npm as well, might take until I get home though. ( On vacation for another week)

**Comment by VibroAxe** [comment]
@mxtommy thanks! enjoy your holiday! @godind should be good to merge now

**Pull request merged**